### PR TITLE
Modelling connection daniel

### PR DESCRIPTION
**Pull request by d-ang1993**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -13,11 +13,13 @@ import projectRouter from "./routes/api/projects";
 import relationshipRouter from "./routes/api/relationships";
 
 var app = express();
+var bodyParser = require("body-parser");
 
 // cors
 app.use(cors());
 
-app.use(bodyParser.json({ limit: '10mb' }))
+// What is this?
```
Comment (wamts) [inline server/app.js]: Its to limit the size of file uploads to the backend.

We might want to add the limit to the front-end too, as right now users don't know if an error happened on the backend (such as during file upload)

**Review by wamts — APPROVED** [review]

File: server/routes/api/projects.js [hunk 2]
```diff
@@ -4,82 +4,186 @@ const User = require("../../models").User;
...
+          entity_id: entity_id
+        }
+      });
+
+      if (!likedEntity) {
+        Liked_Entity.create({
+          user_id: userId,
+          entity_id: entity_id
+        }).then(like => {
+          console.log("like_created");
```
Comment (wamts) [inline server/routes/api/projects.js:239]: include a res.status or res.send here

Comment (d-ang1993) [reply server/routes/api/projects.js:239]: ;] yup

File: server/app.js (hunk 1 above)
Comment (d-ang1993) [reply server/app.js]: Ah I see, I can add the validations to the front end any time

**Comment by d-ang1993** [comment]
closing this pr
resolved changes within a different branch

**Pull request closed without merging**